components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementServiceImpl.java

```diff
@@ -1440,6 +1477,19 @@ public String exportSPApplication(String applicationName, boolean exportSecrets,
         }
         return marshalSP(serviceProvider, tenantDomain);
     }
+    public ServiceProvider exportSP(String applicationName, boolean exportSecrets, String tenantDomain)
```

Comment:
We already have `exportSPApplication`.  I don't see any difference in the functionality compared to that method.